ztrouy/cpyou
# Feature - Initial Branding

### Purpose
To provide a logo and new primary color for use throughout the website

### Files Changed
- Added logo vector in new `LogoBlue.svg` and `LogoWhite.svg` vector files
- Updated nav button for Home view to use LogoWhite instead of type in `NavBar.jsx`
- Created basic theme to change primary color throughout website in new `baseTheme.jsx`
- Wrapped return in MUI ThemeProvider to provide the theme throughout the website in `App.jsx`
- Updated spacing around buttons, and updating text content rendering in `CommentModal.jsx` and `DeleteModal.jsx`
- Updated rendering to have a max width of 800px to match rest of the website in `BuildDetails.jsx`

### To Test
Fetch, setup, and run according to project README, then confirm the following
- Navbar, buttons, and input highlights throughout website are new deeper blue color
- Nav button for Home view is CPYou logo
- Build Details view content no longer extends wider than content throughout rest of website

closes #49
